websites/C/Creepypasta/dist/metadata.json

```diff
@@ -6,10 +6,10 @@
   },
   "service": "Creepypasta",
   "description": {
-    "en": "Get great PC and Mac games on Origin. Play the latest RPGs, shooters, Sims games & more."
+    "en": "Welcome to Creepypasta.com! Scaring you since 2008 with paranormal stories and creepy original horror fiction. Sleep well...."
```

Comment:
Quadruple dots?
```suggestion
    "en": "Welcome to Creepypasta.com! Scaring you since 2008 with paranormal stories and creepy original horror fiction. Sleep well..."
```

Reply:
It's written like that in the official description 😔
![image](https://user-images.githubusercontent.com/44418429/149599440-1775bcce-1bc1-4395-8a06-f1aeb9e74e75.png)